scripts/export_esm.py

```diff
@@ -0,0 +1,49 @@
+import json
```

Comment:
It's good practice to start Python files with a shebang line that clarifies what interpreter should be used:
```suggestion
#!/usr/bin/env python3

import json
```